File: benchmarks/leibniz/leibniz.js
function calc_pi(n) {
    let result = 0;
    for (let i = 0; i < n; i += 1) {
        result += Math.pow(-1, i) / (2 * i + 1);
    }
    return result * 4;
}

console.log(calc_pi(50_000_000));

File: benchmarks/leibniz/leibniz.rs
fn calc_pi(n: u32) -> f64 {
    let mut result = 0.0;
    for i in 0..n {
        result += i32::pow(-1, i) as f64 / (2 * i + 1) as f64;
    }
    result * 4.0
}

fn main() {
    println!("{}", calc_pi(50_000_000));
}
